demo-opensuse/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM opensuse
+MAINTAINER Shikher Verma
+
+RUN cd home/
+RUN zypper --non-interactive install git python-pip gcc python-devel libffi-devel libopenssl-devel vim build sudo osc wget
+RUN zypper --non-interactive install -t pattern devel_C_C++
```

Comment:
Is this required for the final functionary? Maybe this could be a cool place to try out [`multi-stage builds`](https://docs.docker.com/engine/userguide/eng-image/multistage-build/#before-multi-stage-builds) to save on the final container size.